docker/fullnode/README.md

```diff
@@ -30,6 +29,29 @@ Get the latest version of the Sui DevNet genesis [genesis.blob](https://github.c
 ```wget https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob```
 
 
+### docker compose 
+
+Download the latest version of docker compose
+
+```wget https://github.com/MystenLabs/sui/blob/main/docker/fullnode/docker-compose.yaml```
```

Comment:
this is a link to a yaml file in the Sui repo, I think it just gets the yaml file vs install Docker Compose? There is an instruction in Prereqs to first install it.

Reply:
Yesn, its in Prereqs, but it is not specified that it must be downloaded.